lib/logstash/outputs/elasticsearch.rb

```diff
@@ -50,114 +49,18 @@
 # Keep in mind that a connection with keepalive enabled will
 # not reevaluate its DNS value while the keepalive is in effect.
 class LogStash::Outputs::ElasticSearch < LogStash::Outputs::Base
-  attr_reader :client
+  require "logstash/outputs/elasticsearch/http_client"
```

Comment:
I too am against lazy loads.

This isn't actually a lazy load, its more of an ordered load. This will be loaded when the file is loaded. The reason its here, rather than top of file, is that doing this at the top of the file breaks unless all modules look like `module LogStash; module Outputs; module ElasticSearch < LogStash::Outputs::Base`.

The subclassing belongs here, not there, so this was the best workaround I was able to find.